### ISSUE TITLE
SA1113 not reported in BaseConstructorInitializer

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1654]
### Source code

```csharp
abstract class BaseClass
{
    protected BaseClass(int x, int y) { }
}

class ClassName : BaseClass
{
    protected ClassName(int x, int y)
        : base(
            x
            , y)
    {
    }
}
```

### Expected

SA1113 reported on the `,` before `y`.

### Actual

No warning reported.

**Comment by vweijsters** [comment]
Grabbing this
